src/assets/styles/styles/custom/_workers_page.scss

```diff
@@ -0,0 +1,20 @@
+@import "variables";
+
+.workers-table {
```

Comment:
To jest najmądrzejszy sposób w jaki udało mi się to ugryźć. Jest trochę zamieszania ze stylami w tej tabeli, ponieważ raz korzystam z cssów z materiala, a raz jak tutaj z sccsa

Reply:
Mi się, wydaję, że było by lepiej żeby style były w jednym miejscu (jeżeli da sie to zrobiić), bo jeżeli my będziemy chcieli to kiedyś zmienić, to będziemy od razu widzieć co mmy musimy zmieniać i nie będziemy mielis jakichś tajenmyczyc styli które musimy nadpisywać za pomocą `import`

Reply:
No nie podoba mi się to za bardzo, ale nie znaleźliśmy lepszego na razie. Wiem, że w przyszłości to może być problem, ale na razie się tym nie zajmujmy. Zróbmy to iteracyjnie i po pierwszym milestonie będzie wielki refactoring #can't#wait XD